supervisor/docker/interface.py

```diff
@@ -377,6 +377,31 @@ async def run(self) -> None:
         """Run Docker image."""
         raise NotImplementedError()
 
+    async def _run(self, **kwargs) -> None:
+        """Run Docker image with retry inf necessary."""
+        if await self.is_running():
+            return
+
+        # Cleanup
+        await self.stop()
+
+        # Create & Run container
+        try:
+            docker_container = await self.sys_run_in_executor(
+                self.sys_docker.run, self.image, **kwargs
+            )
+        except DockerNotFound as err:
+            # If image is missing, capture the exception as this shouldn't happen
+            # Try to keep things working for user by pulling image and retrying once
+            capture_exception(err)
+            await self.install(self.version)
```

Comment:
@pvizeli what do you mean by "Let's not do that" exactly?

IMHO adding the sentry capture is fine here, but maybe remove the automatic install for now, until we have a good understand in which cases the image can be missing and they warrant adding it automatically?

```suggestion
```

Reply:
right, that is what I mean